Squert styling 2.0

innovate-support opened the pull request:
Here's my Squert 2.0 style mods.  I included the original files as well, but renamed them and added a Style selector so the original layout is still accessible.  Thanks, Kris Springer

https://drive.google.com/file/d/0BxO-LRVGeXnnYTJuM0FRN3JDbU0/view?usp=sharing

Screenshot for reference.
-----------------------------
![style 2 0](https://cloud.githubusercontent.com/assets/20424476/21330397/176b4ba0-c5f2-11e6-9a15-2b6f7abc7a87.png)

dougburks committed:
update README

dougburks committed:
update version and year

dougburks committed:
pivot to capme for transcript

dougburks committed:
add securityonion scripts and sql

dougburks committed:
add URL lookups to securityonion_update.sql

dougburks committed:
fix OSSEC alerts

dougburks committed:
fix several issues

dougburks committed:
add hash to object_mappings table

dougburks committed:
populate empty hash fields

dougburks committed:
Squert: OSSEC HIDS alerts display NIDS rules #958

dougburks committed:
initialize srcd, dstd, and alld in callback.php

dougburks commented:
Hi @innovate-support ,

Per our discussion at https://groups.google.com/d/topic/security-onion/VIwetmcW4zY/discussion, did you mean to submit this pull request to https://github.com/Security-Onion-Solutions/securityonion-squert instead?

Also, when submitting pull requests, please submit the actual files rather than a link to Google Drive.

Thanks.

innovate-support commented:
I'd be more than happy to comply if I could figure out how to submit the files properly.  I go to https://github.com/Security-Onion-Solutions/securityonion-squert and click the 'New pull request' button get the page where I submitted the previous link to Google drive.  I sent it that way because it wouldn't let me upload any files.

I also installed the Git desktop software, cloned the Squert master, created a fork, committed my edits, and then clicked 'Pull request', but it gives a permissions error since I don't have write permissions to your repo.   Clicking 'Publish' also gives a permissions error.  In short, I'd like to submit the files in your preferred medium if I could figure out how to.  So I gave up and just sent you a Google Drive link to a zip file.

innovate-support commented:
I've also got the files here.
https://github.com/innovate-support/squert

dougburks committed:
Update README.md

dougburks committed:
Merge pull request #1 from Security-Onion-Solutions/dougburks-patch-1

Update README.md

petiepooo committed:
Avoid hard loop when file unavailable

When the download is not available, this script gets stuck in a hard loop retrying, filling up both logfiles and bandwidth to a proxy.  This patch neuters the retry attempt by just continuing on.

dougburks committed:
Squert: error when removing comment #1066

dougburks committed:
Merge branch 'master' of https://github.com/Security-Onion-Solutions/securityonion-squert

dougburks committed:
update README.md

dougburks committed:
bump version to 1.6.4

dougburks committed:
Squert: ip2c avoid hard loop when file unavailable #1067

dougburks committed:
Merge pull request #3 from petiepooo/patch-1

Avoid hard loop when file unavailable

dougburks committed:
Squert: comment search not working #1119

dougburks committed:
fix error in times function

weslambert committed:
validate filter 'name' field

weslambert committed:
remove check for built-in(s)

dougburks committed:
Merge pull request #4 from weslambert/master

validate filter 'name' field

dougburks committed:
improve input validation and output filtering

weslambert committed:
add check for notes and url

dougburks committed:
Merge pull request #5 from weslambert/master

add check for notes and url

dougburks committed:
merge and adjust comment

dougburks committed:
improve calls to clicat

dougburks committed:
version 1.7.0

dougburks committed:
add row for Bro agent

dougburks committed:
version 1.7.1

dougburks committed:
1.8.0 - update for 16.04

dougburks committed:
Issue 1169: Squert: remove search link from context menu

dougburks committed:
Issue 1259: Squert: turning grouping off results in no alerts

dougburks committed:
Squert: Priority counts incorrect #1277

dougburks committed:
securityonion-squert: increase group_concat_max_len Security-Onion-Solutions/security-onion#1602

dougburks committed:
Squert: improve consistency of username handling Security-Onion-Solutions/security-onion#1643
